docs/api-reference/core/animation-loop.md

```diff
@@ -132,6 +131,8 @@ The callbacks `onInitialize`, `onRender` and `onFinalize` that the app supplies
 | `_offScreen` | `Boolean` | (**experimental**) If the animation loop is rendering to an OffscreenCanvas. |
 | ...       | Any fields in the object that was returned by the `onInitialize` method. |
 
+### Frame timers
+* The animation loop tracks GPU and CPU render time of each frame the in member properties `cpuTime` and `gpuTime`. If `gpuTime` is set to `-1`, then the timing for the last frame was invalid ("disjoint") and should not be used.
```

Comment:
Maybe a comment about what disjoint means (my "hand wavy" sense is that most common cause is that the laptop went into sleep mode).

Reply:
I'll admit I never looked into this. Turns out it's if the GPU clock frequency changes mid-frame, e.g. if the GPU starts being throttled mid-frame 
https://docs.microsoft.com/en-us/windows/desktop/api/d3d11/ns-d3d11-d3d11_query_data_timestamp_disjoint